internal/manifest/list.go

```diff
@@ -65,6 +69,32 @@ type ListUpdate struct {
 	MediaType string
 }
 
+type ListOp int
+
+const (
+	ListOpInvalid ListOp = iota + 1
```

Comment:
The idea of explicitly reserving `ListOpInvalid` is that is the _zero_ element, not `1`. We don’t need to treat 1 specially. With `listOpInvalid = iota`, callers that forget to initialize the field would get a known-invalid value.

Also this enum value can be private; we make no external promises about what using it would do.

Reply:
Done.